SECURITY.md

```diff
@@ -18,17 +18,17 @@ If it is something that can be handled by a param change, discuss it on Discord
 
 For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
 
-If it is not triaged in a couple of days, feel free to tag `@the-frey` or `@jakehartnell`.
+If it is not triaged in a couple of days, feel free to tag `@the-frey` or `@ `.
```

Comment:
rm frey? other at?

also we need a security@juno email

Reply:
whoops, fixing